lib/gli/monkeypatches/with_indifferent_access.rb

```diff
@@ -0,0 +1,17 @@
+# adapted from https://gist.github.com/Integralist/9503099
+class Object
```

Comment:
Yes, that's fair.  It hadn't occurred to me that a GLI project might also have ActiveSupport loaded.